lineapy/utils/lineabuiltins.py

```diff
@@ -258,6 +262,29 @@ def l_unpack_ex(
     return [*before_list, middle_list, *after_list]
 
 
+@register
+def l_import(
+    name: str, base_module: types.ModuleType = None
+) -> types.ModuleType:
+    """
+    Imports and returns a module. If the base_module is provided, the module
+    will be a submodule of the base.
+
+    Marks the `base_module` as modified if provided.
```

Comment:
I don't think the marking is done in `l_import` function def, but rather the yaml annotations. If I'm correct, please clarify in the doc string.

Reply:
tried to reword in the next commit.